### PR TITLE
Fixes #10811 - merged settings from plugin templates

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

**Comment by tstrachota** [comment]
@ShimShtein is `app/controllers/foreman_deployments/create_resources_controller.rb` added by intention? You're adding it also in https://github.com/theforeman/foreman_deployments/pull/4

Otherwise it LGTM.

**Comment by ShimShtein** [comment]
@tstrachota , I will rebase the commit as soon as #4 will be merged

File: foreman_deployments.gemspec [hunk 1]
```diff
@@ -2,17 +2,17 @@ require File.expand_path('../lib/foreman_deployments/version', __FILE__)
...
-  s.description = "Foreman Deployments"
+  s.author     = 'ForemanDeployments team'
+  s.email       = 'foreman_deployments@example.com'
+  s.homepage    = 'https://github.com/theforeman/foreman_deployments'
+  s.summary     = 'Foreman Deployments'
+  s.description = 'Foreman Deployments'
 
-  s.files      = Dir["{app,config,db,lib}/**/*"] + ["LICENSE", "Rakefile", "README.md"]
-  s.test_files = Dir["test/**/*"]
+  s.files      = Dir['{app,config,db,lib}/**/*'] + ['LICENSE', 'Rakefile', 'README.md']
```
Comment (tstrachota) [inline foreman_deployments.gemspec:14]: The license file is actually named `LICENSE.txt`. Bundler complains about missing file.

**Comment by ShimShtein** [comment]
@tstrachota renamed the file, as it appears in the template repository

**Comment by tstrachota** [comment]
I think we can merge this one when we remove the controller file.

**Comment by ShimShtein** [comment]
@tstrachota controller gone :)

**Comment by tstrachota** [comment]
Cool, thanks @ShimShtein. Merging...

**Pull request merged**